docs/user_guide/transformation/PowerTransformer.rst

```diff
@@ -5,66 +5,349 @@
 PowerTransformer
 ================
 
-The :class:`PowerTransformer()` applies power or exponential transformations to numerical
-variables.
+Power transformations are a family of mathematical functions used to transform numerical variables into a more suitable shape for modeling. The transformation function is typically represented as :math:`x' = x^\\lambda`, where :math:`x` is the original variable and :math:`\\lambda` is the transformation parameter.
```

Comment:
Hey @cmcouto-silva thanks for the PR. Could you please break all lines so that they are around 88 characters max?

just need to press enter at the 88 char height and reorganize the text a bit all throughout the document. Thank you!

Reply:
Will do!